src/field-renderer/FieldRenderer.jsx

```diff
@@ -18,14 +21,28 @@ const FormFieldRenderer = (props) => {
     if (props.handleBlur) { props.handleBlur(e); }
   };
 
+  const { formatMessage } = useIntl();
+
+  const renderLabelWithTooltip = (description) => (
+    <Form.Label isInline>
+      {description}
+      <div className="authn-tooltip-icon">
+        i
+        <div className="authn-tooltip">
+          {formatMessage(messages['tooltip.text'])}
+        </div>
+      </div>
+    </Form.Label>
+  );
+
   switch (fieldData.type) {
     case 'select': {
       if (!fieldData.options) {
         return null;
       }
       formField = (
         <Form.Group controlId={fieldData.name} isInvalid={!!(isRequired && errorMessage)}>
-          <Form.Label isInline>{fieldData.description}</Form.Label>
+          {fieldData.name === 'is_organization_registered' ? renderLabelWithTooltip(fieldData.description) : <Form.Label isInline>{fieldData.description}</Form.Label>}
```

Comment:
Let's make it more readable
```suggestion
          {fieldData.name === 'is_organization_registered' 
            ? renderLabelWithTooltip(fieldData.description) 
            : <Form.Label isInline>{fieldData.description}</Form.Label>}
```

Reply:
Fixed